pkg/tiller/release_server.go

```diff
@@ -244,28 +244,35 @@ func GetVersionSet(client discovery.ServerGroupsInterface) (chartutil.VersionSet
 	return chartutil.NewVersionSet(versions...), nil
 }
```

Comment:
we need server side changes also for this? it means we need to have our own tiller image also

Reply:
Seems likely. It is changing the template rendering process.